Download snippet for Chocolatey includes argument that doesn't work

### URL:

https://nodejs.org/en/download

### Browser Name:

Arc (Chromium)

### Browser Version:

137.0.7151.69

### Operating System:

Windows 11 Version 22H2 (Build 22621.4317)

### How to reproduce the issue:

1. Go to the [download page](https://nodejs.org/en/download)
2. Select Node.js 22.17.0 for Windows using Chocolatey with npm if it isn't automatically selected
3. Copy and paste the snippet for actually installing Node.js, it should be `choco install nodejs-lts --version="22"`
4. Observe the failure

Using the command generated this response:
![Image](https://github.com/user-attachments/assets/13a90a7b-f4ff-46d4-9745-9fff158c591b)

From Chocolatey logs:
```log
2025-07-03 16:12:18,004 22316 [ERROR] - nodejs-lts not installed. The package was not found with the source(s) listed.
 Source(s): 'https://community.chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Version was specified as '22'. It is possible that version
 does not exist for 'nodejs-lts' at the source specified.
Please see https://docs.chocolatey.org/en-us/troubleshooting for more
 assistance.
2025-07-03 16:12:18,011 22316 [WARN ] - 
Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
2025-07-03 16:12:18,011 22316 [INFO ] - 
2025-07-03 16:12:18,012 22316 [ERROR] - Failures
2025-07-03 16:12:18,013 22316 [ERROR] -  - nodejs-lts - nodejs-lts not installed. The package was not found with the source(s) listed.
 Source(s): 'https://community.chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Version was specified as '22'. It is possible that version
 does not exist for 'nodejs-lts' at the source specified.
Please see https://docs.chocolatey.org/en-us/troubleshooting for more
 assistance.
```

If you specify the version as `22.17.0` instead of just `22` it does work. However, it may be better to just use `nodejs` instead of `nodejs-lts`, and specify the full version.

--- a/apps/site/snippets/en/download/choco.bash
+++ b/apps/site/snippets/en/download/choco.bash
@@ -2,7 +2,7 @@
 powershell -c "irm https://community.chocolatey.org/install.ps1|iex"
 
 # Download and install Node.js:
-choco install nodejs-lts --version="${props.release.major}"
+choco install nodejs --version="${props.release.version}"
 
 # Verify the Node.js version:
 node -v # Should print "${props.release.versionWithPrefix}".

